packages/swing-store/src/swingStore.js

```diff
@@ -21,6 +21,7 @@ export function makeSnapStoreIO() {
     tmpName,
     createReadStream: fs.createReadStream,
     createWriteStream: fs.createWriteStream,
+    now: Date.now,
```

Comment:
I think we should pass `performance` instead of `Date`. That's monotonic and what we usually use to measure durations in other places of the code base.